peer/brontide.go

```diff
@@ -1560,11 +1560,11 @@ func messageSummary(msg lnwire.Message) string {
 			msg.PendingChannelID[:], msg.FundingPoint)
 
 	case *lnwire.FundingSigned:
-		return fmt.Sprintf("chan_id=%v", msg.ChanID)
+		return fmt.Sprintf("chan_id=%v", chainhash.Hash(msg.ChanID))
```

Comment:
This only corresponds to the funding TXID if the funding output index is 0.

From a comment on `ChanID`:

```
// ChannelID is a series of 32-bytes that uniquely identifies all channels
// within the network. The ChannelID is computed using the outpoint of the
// funding transaction (the txid, and output index). Given a funding output the
// ChannelID can be calculated by XOR'ing the big-endian serialization of the
// txid and the big-endian serialization of the output index, truncated to
// 2 bytes.
```

So I'm not sure if this is even more misleading? Because you won't get any matches if you grep for the outpoint but the first 30 bytes would actually match...

Reply:
Oh no, that is that micro-optimization to reduce the size of a channel point with two bytes. Missed that completely because I saw it work (I was lucky with output index 0 apparently).

Do we have any options here to make it good?